Content.Client/Options/UI/Tabs/AudioTab.xaml.cs

```diff
@@ -63,9 +65,17 @@ public AudioTab()
         Control.AddOptionCheckBox(CCVars.AdminSoundsEnabled, AdminSoundsCheckBox);
         Control.AddOptionCheckBox(CCVars.BwoinkSoundEnabled, BwoinkSoundCheckBox);
 
+        _admin.AdminStatusUpdated += UpdateAdminButtonsVisibility;
```

Comment:
You need to listen for this in `EnteredTree()` and unsubscribe it in `ExitedTree()`.

Reply:
Done